### PR TITLE
snap: add snap {enable,destory}-classic

### PR DESCRIPTION
**Pull request by mvo5**

### CODE REVIEW

**Comment by zyga** [comment]
Looks good, +1

File: cmd/snap/cmd_classic.go [hunk 1]
```diff
@@ -22,31 +22,30 @@ package main
 import (
 	"fmt"
 
+	"github.com/jessevdk/go-flags"
+
 	"github.com/ubuntu-core/snappy/classic"
 	"github.com/ubuntu-core/snappy/i18n"
-	"github.com/ubuntu-core/snappy/logger"
 	"github.com/ubuntu-core/snappy/progress"
 )
```
Comment (niemeyer) [inline cmd/snap/cmd_classic.go]: `// FIXME: Implement feature via "snap install classic"`

File: cmd/snap/common.go [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+	bs, err := exec.Command("systemctl", "show", "--property=SubState", unitName).CombinedOutput()
+	if err != nil {
+		return false
+	}
+
+	return strings.TrimSpace(string(bs)) == "SubState=running"
+}
+
+// withMutexAndRetry runs the given function with a filelock mutex and provides
+// automatic re-try and helpful messages if the lock is already taken
```
Comment (niemeyer) [inline cmd/snap/common.go]: // FIXME: This function is a left over from the old CLI and a major red flag. No client-side locks should exist.

**Comment by niemeyer** [comment]
LGTM with the two commands.

**Comment by niemeyer** [comment]
(two comments, not commands)

**Comment by pedronis** [comment]
should this be hidden for 16.04 (or on the desktop in general) ? something for next week?

**Comment by mvo5** [comment]
retest this please

**Pull request merged**